### PR TITLE
Load device on boot

### PR DESCRIPTION
**Pull request by xnkevinnguyen**
# Description:
Programatically load device corresponding with command palette via script variable.

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Limitations:

Please describe limitations of this PR

# Testing:

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My code has been formatted with `npm run format` and passes the checks in `npm run check`
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/debuggerCommunicationServer.ts [hunk 1]
```diff
@@ -24,6 +26,8 @@ export class DebuggerCommunicationServer {
         this.simulatorWebview = webviewPanel;
         this.initEventsHandlers();
         console.info(`Server running on port ${this.port}`);
+
+        this.currentActiveDevice=currentActiveDevice
```
Comment (vandyliu) [inline src/debuggerCommunicationServer.ts]: _nit_: add space on both sides of equal sign

File: src/debuggerCommunicationServer.ts [hunk 2]
```diff
@@ -11,10 +11,12 @@ export class DebuggerCommunicationServer {
     private serverHttp: http.Server;
     private serverIo: socketio.Server;
     private simulatorWebview: WebviewPanel | undefined;
+    private currentActiveDevice;
 
     constructor(
         webviewPanel: WebviewPanel | undefined,
-        port = SERVER_INFO.DEFAULT_SERVER_PORT
+        port = SERVER_INFO.DEFAULT_SERVER_PORT,
+        currentActiveDevice:string
```
Comment (vandyliu) [inline src/debuggerCommunicationServer.ts]: _nit_: space after colon

File: src/debuggerCommunicationServer.ts [hunk 3]
```diff
@@ -11,10 +11,12 @@ export class DebuggerCommunicationServer {
     private serverHttp: http.Server;
     private serverIo: socketio.Server;
     private simulatorWebview: WebviewPanel | undefined;
+    private currentActiveDevice;
```
Comment (vandyliu) [inline src/debuggerCommunicationServer.ts]: _nit_: add type

File: src/extension.ts [hunk 4]
```diff
@@ -1137,9 +1140,10 @@ function getWebviewContent(context: vscode.ExtensionContext) {
             </head>
           <body>
             <div id="root"></div>
-            <script>
+            <script >
               const vscode = acquireVsCodeApi();
             </script>
+            <script ></script>
```
Comment (sagarmanchanda) [inline src/extension.ts]: What is this empty block for?

**Review by sagarmanchanda — APPROVED** [review]

File: src/extension.ts (hunk 4 above)
Comment (xnkevinnguyen) [reply src/extension.ts]: Not used! Removed

**Review by vandyliu — APPROVED** [review]
lgtm

**Pull request merged**